ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__multi-nightly.yaml

```diff
@@ -57,6 +57,18 @@ resources:
       cpu: "1"
       memory: 1Gi
 tests:
+- as: aws-ipi-amd-f7-netobserv
```

Comment:
I wonder is it on purpose to cover both arm and arm arches in two netobserv jobs? If netobserv jobs is not arch sensitive, we suggest to run the testing on arm arch to save cost. The same comment on gcp and azure jobs.

Reply:
> ARM jobs are run only on AWS once a month

The `f7` in `aws-ipi-arm-f7-netobserv` means once per 7 days, so the job is running weekly, but not monthly, right?

And in theory, there is no much difference between arm and amd jobs, if you want to cover both arches on purpose, one is once per week, another is once per month, I suggest to run the `f7` job on arm, another is named with `f28` running on amd to save cost. About the frequency setting about `f` field, and how to generate an uniq cron, you can refer to https://github.com/openshift/release/blob/efd66d3ae22fd32abed4f3b2c7921196ea4d02ff/ci-operator/config/openshift/openshift-tests-private/README.md#job-frequency

Reply:
Done

Reply:
@Amoghrd - m6i.2xlarge machine type doesn't exist with ARM arch. Closest available ARM machine type to m6i.2xlarge are m6gd.2xlarge and m7g.2xlarge - were you able to successfully run ARM rehearsal?

@jianlinliu - Based on which ARM machine we go with to match closely for resources we require it's just a difference of $0.02-$0.05/hour (m6gd.2xlarge ~ 0.49, m7g.2xlarge ~ 0.45 vs m6i.2xlarge ~ 0.51 for on-demand instances), I don't think if it would be significant impact on our budget. If it's okay I'd like to keep AMD runs every week and ARM run once per month to catch issues on AMD platform sooner as I presume most of our current customers are on AMD.

Reply:
I can bump up the machines to `m6gd.2xlarge` if that's going to help our tests

Reply:
Also reverted trigger updates, i.e ARM runs monthly and AMD weekly

Reply:
+1 to weekly AMD and monthly ARM